### PR TITLE
Use Main::runStatic in PyBlackoilSimulator

### PR DESCRIPTION
**Pull request by akva2**
Motivation is getting rid of the FLOW_BLACKOIL_ONLY hack.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/flow/Main.hpp [hunk 1]
```diff
@@ -290,7 +288,6 @@ namespace Opm
 
             if (false) {}
 
-#ifndef FLOW_BLACKOIL_ONLY
```
Comment (bska) [inline opm/simulators/flow/Main.hpp:293]: If we're really removing this check then we can also remove the `if (false) {}` check above.

Comment (akva2) [reply opm/simulators/flow/Main.hpp:293]: ah yes.

**Review by bska — COMMENTED** [review]
Very good.  Maybe we'll even be able to move some of the implementation to a `.cpp` file, especially the `run*()` functions.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Very good.  I'll merge once #3682 is in.

**Comment by bska** [comment]
> I'll merge once #3682 is in.

That PR has been merged.  Merging this now as promised.

**Pull request merged**